components/NewRealmWizard/components/NFTCollectionSelector.tsx

```diff
@@ -16,6 +16,7 @@ function ImageWithLoader({ className, ...props }) {
       </div>
       <img
         {...props}
+        src={props.src}
```

Comment:
?

Reply:
...props are not typed and it dont recognize there is props.src

Reply:
ill add htmlInputProps